index.d.ts

```diff
@@ -172,13 +171,6 @@ declare module 'plaid' {
     phone_numbers: Array<PhoneNumber>;
   }
 
-  interface AccountIdentity {
```

Comment:
We still need this schema for the `Asset` related endpoints - I think we can add typings to the effect of `AssetIdentity` 

the address schema has not changed for the assets product, only for everything else (`/accounts` `/identity`).

Reply:
:zap: